src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -144,10 +133,7 @@ def main(context)
   # Why not try the Appwrite SDK?
   #
   # client = Appwrite::Client.new
-  # client
-  #   .set_endpoint('https://cloud.appwrite.io/v1')
-  #   .set_project(ENV['APPWRITE_FUNCTION_PROJECT_ID'])
-  #   .set_key(ENV['APPWRITE_API_KEY'])
+  # client.set_key(context.req.headers['x-appwrite-key']) Set API key; it's automatically provided by Appwrite
```

Comment:
Let's keep the multiline like before so we can make it clear the comment is for the `setKey()` part.